Button/Button.module.less

```diff
@@ -306,7 +307,8 @@
 			.bg {
 				background-color: @agate-button-active-bg-color;
 				background-image: @agate-button-active-bg-image;
-				border: @agate-button-border-width solid @agate-button-border-color;;
+				border: @agate-button-border-width solid @agate-button-border-color;
+				opacity: @agate-button-active-opacity;
```

Comment:
This code breaks the previous button look. Did you intend to do it.

Reply:
yes, the new zeplin spcification for button adds a new opacity for active state for Button. Since ToggleButton is based on Button, I added the fix here
![image](https://user-images.githubusercontent.com/63335068/83724012-409b5800-a648-11ea-9cbf-eff9eb2a6793.png)